test/uk/gov/hmrc/agentuserclientdetails/repositories/FakeJobMonitoringRepository.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentuserclientdetails.repositories
+import com.mongodb.client.result.UpdateResult
+import reactivemongo.bson.BSONObjectID
+
+import java.time.LocalDateTime
+import scala.concurrent.Future
+
+case object FakeJobMonitoringRepository extends JobMonitoringRepository {
```

Comment:
It should be okay to just use a ScalaMock mock for this class.
For the other repositories I kept running into problems trying to mock them as they extended WorkItemRepository and did some init that caused trouble, so I created this manual stub, but maybe this simpler repository could be mocked normally.
(In fact there has to be a way to mock the other repos too without all this boilerplate, but that's something to try another day)